lib/mayaUsd/fileio/translators/translatorSkel.cpp

```diff
@@ -1220,17 +1210,6 @@ bool UsdMayaTranslatorSkel::CreateSkinCluster(
         CHECK_MSTATUS_AND_RETURN(status, false);
     }
 
-    // Connect restMesh.outMesh -> groupParts->inputGeometry
```

Comment:
Those were moved as they are to L1311. I don't think we need a rest pose if we have blendShapes attached to it. We'll use the blendShapes instead